### DISCUSSION TITLE
Problems with Delaunay Traingulation based on multiple Point Sources

### DISCUSSION THREAD

**Discussion by Turbofips** [Universite-Gustave-Eiffel/NoiseModelling#688] [Q&A] [answered]
Hey guys,

I can't figure out what's happening here. I have a small battery park with about 70 noise emitters. When I try to create the Delaunay Grid the results are very strange.
![image](https://github.com/user-attachments/assets/be4170ec-11a7-4774-bff6-5af049ea5e22)
When I reduce the Point Source to just one noise emitter the caluclation works as expected.
![image](https://github.com/user-attachments/assets/385f9133-7b85-4065-9e3f-d275626ad6fe)
Is there a limit of Point Sources I can put into the model?

**Comment by nicolas-f** [comment]
Hi,

As source are inserted into the triangulation you should not put source points/lines near buildings vertex ( at least 2cm from buildings lines/points)

This is delaunay precision issues

You should also remove duplicate source position before feeding delaunay_grid

**Reply by Turbofips** [reply]
Thanks for the answer Nicolas! Very appreciated.
There aren't buildings close to the source points(Next ist about 200m away). However the source points are pretty close to each other:
![image](https://github.com/user-attachments/assets/6fb4da6e-1a2d-4ffb-a900-8fbbd53e2429)
Is there a minimum distance between source points?

**Reply by nicolas-f** [reply, marked as answer]
Yes there should be at least some centimeters between sources points. But if this is an issue, you can just provide an empty source table to Delaunay_Grid.

**Reply by Turbofips** [reply]
All right, thanks! I created the delauney Grid without the source points and have a plausible result.  How does this impact the result?

**Reply by nicolas-f** [reply]
Delaunay grid add a buffer around source points in order to not insert receivers too close to source emission positions and also to increase the resolution around noise emission sources. If you do not add source into the delaunay grid, you should reduce the maximumarea value in order to have a nice density of receivers.
